packages/kbn-apm-config-loader/src/apm_config.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import type { ServiceConfigDescriptor } from '@kbn/core-base-server-internal';
+
+const migrationSchema = schema.object({
+  apm: schema.object({
+    active: schema.maybe(schema.boolean()),
+    serverUrl: schema.maybe(schema.uri()),
+    secretToken: schema.maybe(schema.string()),
+    globalLabels: schema.object({}, { unknowns: 'allow' }),
+  }),
```

Comment:
Hum, looking at 

https://github.com/elastic/kibana/blob/d5fdeefb1c9e6aa8157310cd73430e4cba7fe17f/packages/kbn-apm-config-loader/src/config.ts#L27-L42

Seems like a lot of properties are missing from the schema?

Current implementation will make the config validation fail (and kibana fail to start) if any of these additional config options are specified in the config, given you added schema registration in `Server.setupCoreConfig`.